conf/defaults.yaml

```diff
@@ -163,6 +163,8 @@ supervisor.run.worker.as.user: false
 supervisor.worker.start.timeout.secs: 120
 #how long between heartbeats until supervisor considers that worker dead and tries to restart it
 supervisor.worker.timeout.secs: 30
+#maximium allowed value for topology.worker.timeout.secs, which if set overrides supervisor.worker.timeout.secs
+supervisor.maximum.worker.timeout.secs: 600
```

Comment:
This can be confusing with `supervisor.worker.heartbeats.max.timeout.secs`

Reply:
There is another config `nimbus.task.timeout.secs`. It's the timeout that Nimbus uses to determine if the `executor` heartbeat had timed out or not.